openshift-sdn.spec

```diff
@@ -0,0 +1,141 @@
+#debuginfo not supported with Go
+%global debug_package %{nil}
+%global gopath      %{_datadir}/gocode
+%global osdn_gopath _output/local/go
+%global gopkg_base  github.com/openshift
+%global import_path %{gopkg_base}/openshift-sdn
+%global commit      1d03a6c75a44a638b9f2590b1475a6c980ce829d
+
+Name:           openshift-sdn
+Version:        0.4
+Release:        1%{?dist}
+Summary:        SDN solutions for OpenShift
+License:        ASL 2.0
+URL:            https://%{import_path}
+ExclusiveArch:  x86_64
+Source0:        https://%{import_path}/archive/%{commit}/%{name}-%{version}.tar.gz
+
+BuildRequires:  systemd
+BuildRequires:  golang >= 1.2-7
+
+Requires:       openvswitch >= 2.3.1
+Requires:       bridge-utils
+
+%description
+%{summary}
+
+%package master
+Summary:        Openshift SDN Master
+Requires:       openshift-sdn = %{version}-%{release}
+Requires:       openshift-master
+
+%description master
+%{summary}
+
+%package node
+Summary:        Openshift SDN Node
+Requires:       openshift-sdn = %{version}-%{release}
+Requires:       openshift-node
+Requires:       docker-io >= 1.3.2
+
+%description node
+%{summary}
+
+%prep
+%setup -q
```

Comment:
When extract tar of source0 we will find openshift-sdn-%{commit},
so should be 
`%setup -q -n openshift-sdn-%{commit}`